app.rb

```diff
@@ -121,7 +129,6 @@ def list_rentals
     puts 'Rentals:'
     @rentals.each do |rent|
       puts " Date: #{rent.date} Book: #{rent.book.title} Author: #{rent.book.author}" if rent.person.id == id
-      puts "\n"
     end
   end
```

Comment:
- [ ] I'm sure you touched the rental code during refactoring, After successfully creating a rental, I couldn't see it even after entering the correct person ID.

![rental](https://user-images.githubusercontent.com/53124548/216984135-6f742865-5e93-447e-b52e-8569d84ad2f7.png)